crunchers/cruncher_active_users.h

```diff
@@ -77,6 +74,13 @@ struct ActiveUsersCruncherImpl {
   using user_list_t = std::list<ActiveUser>;
   using user_map_t = std::unordered_map<std::string, typename user_list_t::iterator>;
 
+  void RemoveUsersOutsideTheTimeWindow() {
+    while (!users_list_.empty() && users_list_.back().us + interval_ <= current_us_) {
```

Comment:
I don't quite understand what means the `[ inclusive, exclusive )` here. Do you suggest to exclude from the time window the last event, whose timestamp equals `current_us_`? If not, here is the problem: suppose we generate exactly one event per second and set the `interval_` to `std::chrono::seconds(1)`. Then, I suppose, we expect to see only one event inside the time window at any moment. And it won't be true if the condition is `...  < current_us_`